### PR TITLE
Upcoming mission design

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

**Review by eivindsjovold — COMMENTED** [review]
Why the name change?

**Comment by Eddasol** [comment]
> Why the name change?

With the removal of choice for start time Queue is a better description of the functionality. It it also the name of this section in the figma sketch

File: frontend/src/components/Pages/FrontPage/MissionOverview/NoMissionPlaceholder.tsx [hunk 1]
```diff
@@ -30,12 +30,12 @@ export function NoOngoingMissionsPlaceholder() {
     )
 }
 
-export function NoUpcomingMissionsPlaceholder() {
+export function NoMissionQueuePlaceholder() {
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/NoMissionPlaceholder.tsx:33]: EmptyMissionQue?

File: frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx [hunk 2]
```diff
@@ -79,7 +83,8 @@ export const ScheduleMissionDialog = (props: IProps): JSX.Element => {
                     disabled={props.frontPageScheduleButtonDisabled}
                     ref={anchorRef}
                 >
-                    {Text('Schedule mission')}
+                    <Icon name="add" size={16} />
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx:86]: Maybe we also should use Enums in files we only have one icon?

Comment (Eddasol) [reply frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx:86]: https://github.com/orgs/equinor/projects/85/views/4?pane=issue&itemId=22119947

File: backend/api/EventHandlers/MissionScheduler.cs [hunk 3]
```diff
@@ -32,9 +32,9 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
         {
             while (!stoppingToken.IsCancellationRequested)
             {
-                foreach (var upcomingMission in UpcomingMissions)
+                foreach (var queuedMission in MissionQueue)
                 {
-                    var freshMission = MissionService.ReadById(upcomingMission.Id).Result;
+                    var freshMission = MissionService.ReadById(queuedMission.Id).Result;
```
Comment (eivindsjovold) [inline backend/api/EventHandlers/MissionScheduler.cs:37]: Out of scope of this PR - But I would rename this variable to a more descriptive name.

**Review by eivindsjovold — APPROVED** [review]
LGTM

**Pull request merged**